server/src/main/java/org/elasticsearch/transport/TransportResponse.java

```diff
@@ -21,15 +20,6 @@ public abstract class TransportResponse extends TransportMessage {
      */
     public TransportResponse() {}
 
-    /**
-     * Constructs a new transport response with the data from the {@link StreamInput}. This is
-     * currently a no-op. However, this exists to allow extenders to call <code>super(in)</code>
-     * so that reading can mirror writing where we often call <code>super.writeTo(out)</code>.
-     */
-    public TransportResponse(StreamInput in) throws IOException {
-        super(in);
```

Comment:
Shouldn't the similar ctor be removed from TransportMessage?

Reply:
I did that here :) https://github.com/elastic/elasticsearch/pull/120016/files#diff-e0291449c744ed09e781ca6dd31eb84132ee7d81c9e83d4fab507cc4091d8a2bL42 or do you mean something else?